articles/introduction-to-kotlin-flows/index.md

```diff
@@ -0,0 +1,109 @@
+### Introduction
+Suspending functions and/or coroutines launched with the `async` builder return single values. These values can be strings, integers, boolean, or even lists. But, what if we want to return multiple asynchronously computed values? Take the case of loading a large list from a database. Using the default coroutine builders will return the whole list at once. However, this will take long thus making a program slow. The best approach to this case would be to return single values after computation rather than the whole list. Well, how can we do this? That's where flows come in.
+
+### Prerequisites
+To follow through with this tutorial, you will need to:
+  1. Have IntelliJ IDEA installed.
+  2. Have a basic understanding of [Kotlin](/engineering-education/kotlin-collections/) programming language.
+  3. Have a basic understanding of coroutines. Check this [article](/engineering-education/introduction-to-kotlin-coroutines/) to get started.
+
+### What is a flow
+A flow is a stream of multiple, asynchronously computed values. Flows emit values as soon as they are done computing them. A flow consists of a producer and a consumer. As the names suggest, a producer emits values while the consumer receives the values. 
+
+Let's see how we can create and use flows in a kotlin program.
+
+### Step 1 — Creating a Kotlin project
+In this step, we are going to create a Kotlin console project managed by Gradle.
+
+Open IntelliJ and select `New Project`. On the next window, select kotlin, console application. Choose the project JDK, download one if none is installed. 
+
+![New project](/engineering-education/introduction-to-kotlin-flows/new-project.png)
+
+Give the project a name and click next. Leave the next screen to default settings and click finish.
+
+Wait for the project build to finish.
+
+Flows are built on top of coroutines. So, open the `build.gradle` file and add the following dependency.
+```Gradle
+implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.2")
```

Comment:
I see you are using Kotlin Gradle DSL. 
This is not an issue but, maybe next time you provide a Groovy option too since most developers still use it.

The only difference is the brackets. But apparently, "It's the small things..." :joy:

Reply:
Noted😂